### ISSUE TITLE
filesupload.org

### ISSUE THREAD

**Issue by NanoMeow** [NanoMeow/QuickReports#3052]
### Basic Information

**Test link:** `https://filesupload.org/c12e89b311b01777?unlock`
**Category:** ads
**Reported from:** Canada
**Template version:** 2

### User Environment

**Browser:** Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.17 Safari/537.36
**Extension:** Nano Adblocker 1.0.0.135

### Additional Message

First click of "download now" button result in redirection to ads. Improperly blocked popunder?

**Comment by jspenguin2017** [comment]
@okiehsch

**Comment by okiehsch** [comment]
I get the file.

**Comment by jspenguin2017** [comment]
On the first click?

**Comment by okiehsch** [comment]
Yes. I get redirected to `https://multinews.me/produit/rapidgator` and the download starts immediately.
I don't see any ads the multinews site is blank on my end.

**Comment by ghajini** [comment]
@okiehsch I never get the file....clear cookies +data /avoid incognito mode, change ip, then try. ...

**Comment by ghajini** [comment]
This is indeed fixed in `https://github.com/timmyRS/Universal-Bypass/issues/933`
2 cases like this
at `filesupload.org` and `multiup.org` so this much enough for gorhill to consider fixing these issues

**Comment by ghajini** [comment]
They removed that annoyance 2 days ago, so nonissue now....I will report back when it reappears..
Also https://github.com/NanoMeow/QuickReports/issues/2096 not fixed yet....

**Comment by jspenguin2017** [comment]
Feel free to contribute.
